SparkPost/2web2ui
# SA-1328: Summary Report - Hibana Copy

### What Changed
 - Made a copy of the summary report page
 - Label changed to Analytics report

### How To Test
 - On account with Hibana flag enabled, check summary page
 - Verify it's copy of page by checking title. Should be `Analytics Report`
